config/sync/core.entity_form_display.paragraph.grid_item.default.yml

```diff
@@ -21,16 +21,17 @@ content:
     settings:
       title: Paragraph
       title_plural: Paragraphs
-      edit_mode: open
+      edit_mode: closed_expand_nested
```

Comment:
This did the opposite of what needed to happen, the entire paragraph should be closed by default